src/components/VideoPreview.tsx

```diff
@@ -78,6 +85,7 @@ export const VideoPreviewPublisher: React.FC<VideoPreviewWPublisherProps> = ({ i
       duration={video?.duration}
       views={video?.views}
       thumbnailUrl={thumbnailPhotoUrl}
+      hasThumbnailUploadFailed={hasThumbnailUploadFailed}
```

Comment:
what if the video upload failed, which is more common?
Generally,  it's a bigger problem, because the design is not clear about it. After the transaction is done we are adding the video to MyVideosView, but it's not available yet, it still needs to be uploaded. 
We can even go to the URL where is located(but we can't play it).
Maybe we should add a tab - `videos that's need to be uploaded` or smth(bad name for a tab I know :P)? 

Just want to raise a subject here, I don't think, it's not required to do this in this particular PR